react/components/Table/index.js

```diff
@@ -439,8 +443,8 @@ Table.propTypes = {
     texts: PropTypes.shape({
       secondaryActionsLabel: PropTypes.string.isRequired,
       rowsSelected: PropTypes.func.isRequired,
-      selectAll: PropTypes.string.isRequired,
-      allRowsSelected: PropTypes.func.isRequired,
```

Comment:
Do you need to remove this `isRequired`?

Reply:
This was actually a sneaky bug fix 😅  I made them optional in another PR but didn't change it here. Now it keeps alerting `selectAll is required!!!!!1!!one`. But it isn't

Reply:
Got you :)

I believe you could leave this change off this PR, and git will add those back when you merge or rebase the PR to master.